src/main/java/de/mossgrabers/convertwithmoss/format/yamaha/ysfc/YamahaYsfcCreator.java

```diff
@@ -212,24 +231,36 @@ private static void storeMultisamples (final List<IMultisampleSource> multisampl
                         throw new IOException (ex);
                     }
 
-                    final byte [] data = dataChunk.getData ();
+                    final byte[] data = dataChunk.getData();
                     final boolean isStereo = numberOfChannels == 2;
 
-                    for (int channel = 0; channel < numberOfChannels; channel++)
-                    {
-                        keybankList.add (createKeybank (sampleNumber, zone, formatChunk, numSamples));
+                    keybankList.add(createKeybank(moxfData, sampleNumber, zone, formatChunk, numSamples));
```

Comment:
Please have a look at this change. 

To my understanding, the keygroup in MOXF can contain stereo files.